packages/core/test/hardhat/data-verification-mechanism/SpamGuard.js

```diff
@@ -483,6 +486,62 @@ describe("SpamGuard", function () {
     assert.equal(await votingToken.methods.balanceOf(account2).call(), toWei("0")); // 0mm.
   });
 
-  it("Correctly sends bond to store if voted down", async function () {});
```

Comment:
two comments:
1) this can be simplified. I don't think you need a) two requests or b) so many votes. if you want to keep both and all the votes then at least check that the price settles on the one that is not deleted.
2) this is the negative test: check that the store gets the bond if voted down. we should also check that the proposer gets it back if voted up in a separate test.

Other than this, looks good!

Reply:
> this can be simplified. I don't think you need a) two requests or b) so many votes. if you want to keep both and all the votes then at least check that the price settles on the one that is not deleted.

This still requires 2 requests since upper bound in spam deletion range cannot be 0 (`signalRequestsAsSpamForDeletion([[0,0]])` would revert).

But agree that this test does not require actually voting on other requests than the spam deletion proposal

Reply:
> this is the negative test: check that the store gets the bond if voted down. we should also check that the proposer gets it back if voted up in a separate test.

Isn't it already covered in `Correct bonds refund` test above?

Reply:
Removed voting on price requests in 0c571c047bca388eebc1448f5e4c517658e9eb52